.github/workflows/test.yml

```diff
@@ -64,16 +78,29 @@ jobs:
       run: cargo doc --workspace --bins
     - name: Run cargo fmt
       run: cargo fmt --all -- --check
-    - name: Build
-      if: always()
```

Comment:
any reason ` if: always()` has been removed?

The idea was that if clippy/rustfmt fails, then we can still try to build/test it (is this useful?)

Reply:
That was an oversight. I'll add that again.